test/i18n.api.js

```diff
@@ -379,14 +389,30 @@ describe('Module API', function () {
           returnedLocales.sort();
           var expectedLocales = ['en', 'de', 'en-GB'];
           expectedLocales.sort();
-          
+
           returnedLocales.length.should.equal(expectedLocales.length);
-          
+
```

Comment:
Sorry about the trailing whitespaces :sweat_smile: my editor insisted in removing them